### PR TITLE
fix build to work on Haiku x86_64

### PR DESCRIPTION
**Pull request by nyssss**
* Link standard C++ libraries `$(STDCPPLIBS)` to LIBS in Makefile.
* Remove default argument for parameter.

### CODE REVIEW

**Comment by scottmc** [comment]
Looking for another set of eyes to review this before merging.

**Review by korli — APPROVED** [review]
All good.

**Pull request merged**
